packages/agent/src/oidc-v2.ts

```diff
@@ -307,17 +260,18 @@ async function createAuthRequest(
   >
 ) {
   // Generate a random state value to associate the authorization request with the response.
-  const { randomStateBytes, randomStateBase64Url } = generateRandomState();
+  const stateBytes = utils.randomBytes(12);
 
   // Generate a random nonce value to associate the ID Token with the authorization request.
-  const nonce = await deriveNonceFromInput(randomStateBytes);
+  const nonceBytes = await Sha256.digest({ data: utils.randomBytes(12) });
```

Comment:
consider just:

```suggestion
  const nonceBytes = utils.randomBytes(32);
```

The `randomBytes()` is called inline, I don't see functional benefit or difference wrapping it with a `Sha256.digest()` call, it add more unanswered questions.

Reply:
We hash because it's part of the specification. This was there before, has nothing to do with inline randombytes? Maybe I'm not understanding the review

Reply:
I am not convinced this is REQUIRED by the specification, can you point me to where in the spec that says this? 

> This was there before

I assume you mean this was part of the code you were given, regardless, it still appears unnecessary to me, you are not reusing the random bytes anywhere, why start with a value with lower entropy when you can just go straight to the higher entropy random value and call it a day.

Reply:
fixed